core/src/main/java/de/danoeh/antennapod/core/feed/LocalFeedUpdater.java

```diff
@@ -32,43 +34,76 @@ public static void startImport(Uri uri, Context context) {
     }
 
     private static void startImportDirectory(Uri uri, Context context) {
+        //create a feed object for this directory
         File f = new File(uri.getPath());
         String dirUrl = uri.toString();
         Feed dirFeed = new Feed(dirUrl, null, "Local directory (" + dirUrl + ")");
-        List<FeedItem> items = new ArrayList<>();
+        dirFeed.setItems(new ArrayList<>()); //this seems useless
```

Comment:
Without this line, there is a NullPointerException in the database (dirty fix :P)